### ISSUE TITLE
Accessibility Issues in Mobile Reader

### ISSUE THREAD

**Issue by JessicaSchouten** [Tizra/Tizra-Customer-Tracker#475]
There are some accessibility issues in the mobile reader. In no particular order:

**1. Search, TOC, and Settings/Preferences icons are not accessible by keyboard**
The home icon is OK.

**2. h4 tags appear on the page with no h1, h2, or h3 preceding them in the page hierarchy**
```
<h4>
Preferences
</h4>
```
```
<h4>
Feedback
</h4>
```
```
<h4>
Table of Contents
</h4>
```

**3. No form label for input (search):**
```
<div class="t-bounds">
<div pubinfo="[object Object]">
<form action=".">
<span class="t-input-text t-button-go">
<input type="search" autocapitalize="off" placeholder="Enter text or page...">
```

**4. There are empty buttons (no value/text):**

This button appears once.
```
<button type="button" class="t-input-clear" style="display: none;">
<span class="icon-close">
</button>
```
This button appears multiple times.
```
<button class="t-button t-icon t-icon-only t-inline" tabindex="-1" label="Expand">
<span class="t-label">
<span class="t-icon-spot icon-larger">
<span class="t-label-text">
</span>
</button>
```

**5. Contrast of left hand nav icons is too low compared to the background**
The contrast ratio is: 2.8:1. Should be 3:1 for non-text items like iconography.

**6. TOC contents are not accessible by keyboard.**
These elements cannot receive focus.

**7. Toggle switches inside of Settings/Preferences menu are not accessible by keyboard.**
These elements cannot receive focus.

**8. Orange color used on TOC and Settings/Preferences is too low contrast.**
Ideally, this might be something handled in properties/left to customers as branding.

The contrast ratio is currently 2.3:1. Ratio must be at least 3:1 for text that is 18pt and larger, or 14pt and larger and bold.
Contrast ratio must be at least 4.5:1 for text that is smaller than 18pt, or smaller than 14pt and bold.
